draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -599,6 +599,14 @@ Here, `eyJhbGci...emhlaUJhZzBZ` represents the SD-JWT and `eyJhb...dYALCGg` repr
 
 # Security Considerations {#security_considerations}
 
+Security considerations in this section help achieve the following the following properties:
+
+* Claim values issued by the Issuer cannot be obtained by the adversary,
+	- if the Holder did not intend to disclose them,
+	- even if the content of the protocol exchange was observed.
```

Comment:
There's not much, if any really, discussion of the second bullet about the exchange being observed in the Security Considerations Section that follows. So I'm not sure it makes sense to call out in this text, which is kinda introducing the Security Considerations.

Reply:
it's [Confidentiality during Transport](https://drafts.oauth.net/oauth-selective-disclosure-jwt/draft-ietf-oauth-selective-disclosure-jwt.html#name-confidentiality-during-tran) in privacy considerations that addresses this second bullet point.

it is tricky because encryption is both security and privacy measure.

Reply:
Yeah, it is tricky. But I don't know about moving that Confidentiality during Transport in into security considerations. I think I'd prefer to just remove that "even if the content of the protocol exchange was observed" part. To keep this new into to the Security Considerations brief and concise.

Reply:
I have read some other RFCs in oauth WG and the text related to encryption usually tends to be under the privacy considerations section. for example, https://datatracker.ietf.org/doc/html/rfc9068#section-6

so I am ok deleting this bullet point. will reformat because now it does not make sense having bullet point.